src/appshell/internal/platform/win/winwindowscontroller.cpp

```diff
@@ -209,6 +240,27 @@ bool WinWindowsController::calculateWindowSize(HWND hWnd, LPARAM lParam, qintptr
     return true;
 }
 
+bool WinWindowsController::initWindowBackgroundColor(HWND hWnd)
+{
+    HDC hdc = GetWindowDC(hWnd);
+
+    QColor bgColor = QColor(uiConfiguration()->currentTheme().values.value(muse::ui::BACKGROUND_PRIMARY_COLOR).toString());
+    COLORREF bgRGB = RGB(bgColor.red(), bgColor.green(), bgColor.blue());
+
+    SetClassLongPtr(hWnd, GCLP_HBRBACKGROUND, (LONG_PTR)CreateSolidBrush(bgRGB));
+    SetBkColor(hdc, bgRGB);
+
+    RECT rect;
+    GetClientRect(hWnd, &rect);
+    rect.bottom = rect.bottom + 100; //! add extra space
+    rect.right = rect.right + 100; //! add extra space
```

Comment:
Why is it necessary, and why 100?

Reply:
For some reason, the client area is returned smaller than the actual dialog area. I couldn’t think of a better solution than to add an extra margin, the size of which I determined experimentally. I'll add a comment